### ISSUE TITLE
ENETDOWN

### ISSUE THREAD

**Issue by wulds** [Dam998/shopify-monitor#20]
When i start the monitor it runs. But after a while i get the error message: "ENETDOWN" how do i solve it?

**Comment by wulds** [comment]
I believe i solved it, it happens when my computer goes into sleepmode. I guess the server goes down then?

**Comment by Brandon1419** [comment]
Yeah... Use digitalocean or something similar

**Comment by msuyat42** [comment]
Hi @wulds were you able to find a way to keep the program running while the computer is on sleepmode?

**Comment by wulds** [comment]
@msuyat42 yes, i had to reinstall mongodb, and had to manually set the data directory.

**Comment by msuyat42** [comment]
@wulds do you mind helping me out with that?

**Comment by wulds** [comment]
@msuyat42 sure add me on discord, ludwig#0808

**Comment by msuyat42** [comment]
@wulds sent you a friend request
